festim/hydrogen_transport_problem.py

```diff
@@ -258,15 +257,14 @@ def create_solver(self):
             self.species[0].solution,
             bcs=self.bc_forms,
         )
-        solver = NewtonSolver(MPI.COMM_WORLD, problem)
-        self.solver = solver
+        self.solver = NewtonSolver(MPI.COMM_WORLD, problem)
+        self.solver.atol = self.settings.absolute_tolerance
+        self.solver.rtol = self.settings.relative_tolerance
+        self.solver.max_it = self.settings.max_iterations
```

Comment:
This might end up clogging the run function up a bit. Plus, with us planning on separating heat transfer solving and hydrogen transport, initialise should be much quicker than before

Reply:
Ok I'm happy leaving it as it is now.